src/components/Books.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Book from './Book';
+// import styles from '../style/Header.module.css';
+
```

Comment:
- [ ] To have a neat codebase, kindly remove this commented-out code that you newly added. Thanks. 👍